tests/unit/test_auth.py

```diff
@@ -112,3 +112,38 @@ def test_add_user_adds_zk_tls_flag(harness: Harness[KafkaCharm]):
             in args["bin_args"]
         ), "--zk-tls-config-file flag not found"
         assert "--zookeeper=" in args["bin_args"], "--zookeeper flag not found"
+
+
+def test_prefixed_acls(harness: Harness[KafkaCharm]):
+    """Checks the requirements for adding and removing PREFIXED ACLs."""
+    with (patch("workload.KafkaWorkload.run_bin_command") as patched_run_bin,):
+        for func in [
+            harness.charm.broker.auth_manager.add_acl,
+            harness.charm.broker.auth_manager.remove_acl,
+        ]:
+            func(
+                username="bilbo",
+                operation="READ",
+                resource_type="TOPIC",
+                resource_name="there-and-back-again",
```

Comment:
**praise**: I just love our LOTR themed tests so much

Reply:
Anything to make writing them less frustrating :P